test/data-structures-test.js

```diff
@@ -57,9 +62,9 @@ describe('Transformations • API Elements', ->
         },
         applicationAst.dataStructures[0]
       )
-    )
+    );
 
-    it('Second element has the correct structure', ->
+    return it('Second element has the correct structure', () =>
```

Comment:
return `it`. This seems like a bad idea and will make it easy for future added tests to be skipped.